### PR TITLE
Restic troubleshooting

### PR DESCRIPTION
**Pull request by weshayutin**
* another troubleshooting guide that we should add
* We get a lot of customer tickets with regards to restic.

### CODE REVIEW

File: docs/restic_troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+    - ReadWriteOnce
+    copyMethod: Direct
+    customCA: {}
+    moverServiceAccount: velero
+    pruneIntervalDays: 1
+    repository: vsb-zg6gg-secret
+```
+
+#### get snapshots
+```
```
Comment (mateusoliveira43) [inline docs/restic_troubleshooting.md:356]: I would separate commands and outputs for each part

For example: here something like

For a repository located at `<REPOSITORY-URL>`, run
```
restic --cache-dir /tmp/.cache -r <REPOSITORY-URL> snapshots
```
After typing your repository password, you should get something like
```
repository 83b7f53a opened (version 2, compression level auto)
ID        Time                 Host        Tags        Paths
------------------------------------------------------------
ab60e48b  2023-06-20 21:04:41  volsync                 /data
------------------------------------------------------------
1 snapshots
```

Comment (weshayutin) [reply docs/restic_troubleshooting.md:356]: ++ true, that was lazy of me.

File: docs/restic_troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+  maintenanceFrequency: 168h0m0s
+  repositoryType: restic
+  resticIdentifier: s3:<REPOSITORY-URL>/<BUCKET>/velero/restic/mysql-persistent
+  volumeNamespace: mysql-persistent
+status:
+  lastMaintenanceTime: "2023-06-19T19:35:40Z"
+  phase: Ready
+```
+
+## retic repo password
```
[the remark this answers is not in the document]
Comment (weshayutin) [reply docs/restic_troubleshooting.md]: thank you!

File: docs/restic_troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+metadata:
+  creationTimestamp: "2023-06-14T17:43:37Z"
+  name: velero-repo-credentials
+  namespace: openshift-adp
+  resourceVersion: "22449264"
+  uid: b75d5f8c-9263-445e-b1a3-167a95c07cdf
+type: Opaque
+
+[whayutin@thinkdoe SETUP]$ echo "c3RhdGljLXBhc3N3MHJk" | base64 
+YzNSaGRHbGpMWEJoYzNOM01ISmsK
```
[the remark this answers is not in the document]
Comment (weshayutin) [reply docs/restic_troubleshooting.md]: Agree :) thank you!

File: docs/restic_troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,363 @@
+# velero restic troubleshooting tips
+
```
Comment (shubham-pampattiwar) [inline docs/restic_troubleshooting.md:2]: Maybe add a sub heading of Velero restic FS copy

**Review by shubham-pampattiwar — APPROVED** [review]

File: docs/restic_troubleshooting.md (hunk 4 above)
Comment (kaovilai) [reply docs/restic_troubleshooting.md:2]:
```suggestion
## Velero restic FS copy
<!--TODO: -->
```

**Pull request merged**